[BUG] Flaky test_multiple_apps_same_component on Rolling CI

# Bug report

### Steps to reproduce
1. Push to main - triggers CI on Rolling (ubuntu:noble)
2. `test_entity_model_runtime` integration test runs
3. `test_multiple_apps_same_component` intermittently fails with 404 for `controller` app

Example failure: https://github.com/selfpatch/ros2_medkit/actions/runs/23902492289/job/69702495917

### Expected behavior

All demo apps are discovered before tests run. `GET /apps/controller` returns 200.

### Actual behavior

`GET /apps/controller` returns 404 (entity-not-found). The `controller` app (from `demo_light_controller` node at `/body/lights/controller`) has not been discovered yet when the test executes.

```
FAIL: test_multiple_apps_same_component
AssertionError: 404 != 200 : GET /apps/controller returned 404
```

Root cause: the test queries `demo_apps = ['temp_sensor', 'actuator', 'lidar_sensor', 'controller']` but the class's `REQUIRED_APPS` only waits for `{'temp_sensor', 'rpm_sensor', 'actuator', 'lidar_sensor'}`. The `controller` app is not in the wait set, so on slower CI runners (Rolling) it may not be discovered in time.

### Environment
- ros2_medkit version: main (b2c04dd)
- ROS 2 distro: Rolling (reproduces there, Jazzy/Humble pass)
- OS: ubuntu:noble (CI)

### Additional information

Fix: add `'controller'` to `REQUIRED_APPS` and bump `MIN_EXPECTED_APPS` to 9 to match the actual demo node count. Also add `'body'` to `REQUIRED_FUNCTIONS` since the test implicitly depends on nodes in the `/body` namespace.

--- a/src/ros2_medkit_integration_tests/test/features/test_entity_model_runtime.test.py
+++ b/src/ros2_medkit_integration_tests/test/features/test_entity_model_runtime.test.py
@@ -46,7 +46,7 @@ def generate_test_description():
 class TestEntityModelRuntime(GatewayTestCase):
     """Verify the SOVD entity model in runtime_only discovery mode."""
 
-    MIN_EXPECTED_APPS = 9
+    MIN_EXPECTED_APPS = len(ALL_DEMO_NODES)
     REQUIRED_FUNCTIONS = {'powertrain', 'chassis', 'body', 'perception'}
     REQUIRED_APPS = {'temp_sensor', 'rpm_sensor', 'actuator', 'lidar_sensor', 'controller'}
 

